server/utils/azureAppInsights.ts

```diff
@@ -21,9 +21,7 @@ function version(): string {
 }
 
 export function initialiseAppInsights(): void {
-  // Loads .env file contents into | process.env
-  config()
-  if (process.env.APPINSIGHTS_INSTRUMENTATIONKEY) {
+  if (process.env.APPLICATIONINSIGHTS_CONNECTION_STRING) {
```

Comment:
this was using incorrect env var name, don't quite know how it was working previously